sk-csharp-chatgpt-plugin/azure-function/Models/AiPluginSettings.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace Models;
+
+#pragma warning disable CA1056
+public class AiPluginSettings
+{
+    public string SchemaVersion { get; set; } = "v1";
+    public string NameForModel { get; set; } = string.Empty;
+    public string NameForHuman { get; set; } = string.Empty;
+
+    public string DescriptionForModel { get; set; } = string.Empty;
+
+    public string DescriptionForHuman { get; set; } = string.Empty;
+
+    public AuthModel Auth { get; set; } = new AuthModel();
+
+    public ApiModel Api { get; set; } = new ApiModel();
+
+
```

Comment:
Pretty print this class

Reply:
I'm not sure how to do that; is there a setting I should be using in VS Code?